src/main/kotlin/lotto/dto/RankedTickets.kt

```diff
@@ -0,0 +1,15 @@
+package lotto.dto
+
+import lotto.controller.TicketsEvaluator
+import lotto.domain.WinningCombination
+
+data class RankedTickets(val purchaseResult: PurchaseResult, val winningCombination: WinningCombination) {
+    val ranked = TicketsEvaluator.runWith(purchaseResult.totalTickets, winningCombination)
+    val totalRate = getTotalRate()
+
+    private fun getTotalRate(): Double {
+        val totalPrize = ranked.sumOf { it.winningMoney }
+        val totalSpent = purchaseResult.userPurchase.amount
+        return totalPrize.toDouble() / totalSpent
+    }
+}
```

Comment:
nooooooooooo. u need check about this getter method!
It makes compilation error 👿